#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-suspend2-note.dpatch by martin f. krafft <madduck@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add note for kernel-patch-suspend2.

@DPATCH@
diff -urNad hibernate-1.91~/suspend2.conf hibernate-1.91/suspend2.conf
--- hibernate-1.91~/suspend2.conf	2006-06-05 13:20:32.000000000 +0200
+++ hibernate-1.91/suspend2.conf	2006-06-05 13:41:10.440408368 +0200
@@ -2,6 +2,9 @@
 #
 # NOTE: Suspend2 is an improved version of suspend-to-disk which currently
 #       requires patching your kernel. For more information, see www.suspend2.net
+#
+#       For the Debian patch, please see
+#       http://packages.debian.org/kernel-patch-suspend2
 # 
 #       If you do not wish to patch your kernel but still be able to suspend to
 #       disk, see disk.conf instead.
